share/completions/sad.fish

```diff
@@ -0,0 +1,9 @@
+complete sad    -s 0 -l read0                    -d 'Use \0 as stdin delimiter'
```

Comment:
This formatting will be completely obliterated the next time we `fish_indent` this file.

It's not worth it to keep alignment.

Reply:
okay, I ran `fish_indent` on it.